src/Php84/composer.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "symfony/polyfill-php84",
+    "type": "library",
+    "description": "Symfony polyfill backporting some PHP 8.4+ features to lower PHP versions",
+    "keywords": ["polyfill", "shim", "compatibility", "portable"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Nicolas Grekas",
+            "email": "p@tchwork.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=7.1"
```

Comment:
Is this correct? just asking

Reply:
PHP 8.3 polyfill has the same constraint. I don't immediately see why we have to raise the minimal PHP version, but we can increase this if we have to when we add the individual polyfill classes/functions.